packages/salesforcedx-sobjects-faux-generator/src/describe/sObjectDescribe.ts

```diff
@@ -222,7 +223,8 @@ export class SObjectDescribe {
       headers: {
         'Content-Type': 'application/json',
         Accept: 'application/json',
-        Authorization: `OAuth ${this.accessToken}`
+        Authorization: `OAuth ${this.accessToken}`,
+        'User-Agent': 'salesforcedx-extension'
```

Comment:
Please remove this per our discussion. It's best to not simple introduce a random user-agent.

Reply:
It needs to remain until the 210/patch is released and verified that it works - otherwise we will get random failures.    It can be removed once the 210/patch is fully available.